pallets/nfts/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+authors.workspace = true
+description = "Fork of FRAME NFTs pallet (polkadot v1.15.0)"
+edition.workspace = true
+homepage = "https://substrate.io"
+license.workspace = true
+name = "pallet-nfts"
+readme = "README.md"
+repository.workspace = true
+version = "31.0.0"
+
+
+[package.metadata.docs.rs]
+targets = [ "x86_64-unknown-linux-gnu" ]
+
+[dependencies]
+codec = { workspace = true }
+enumflags2 = { workspace = true }
+log = { workspace = true }
+scale-info = { features = [ "derive" ], workspace = true }
+
+# Substrate
+frame-benchmarking = { optional = true, workspace = true }
+frame-support.workspace = true
+frame-system.workspace = true
+sp-core.workspace = true
+sp-io.workspace = true
+sp-runtime.workspace = true
+
+[dev-dependencies]
+pallet-balances = { default-features = true, workspace = true }
+sp-keystore = { default-features = true, workspace = true }
```

Comment:
```suggestion
pallet-balances.workspace = true
sp-keystore.workspace = true
```

Reply:
Why do we want to change it? This will cause errors.

Reply:
Because in the main Cargo.toml file it is already defined with `default-features = false` so you don't have to again. Weird that it is causing an error.

Reply:
@Daanvdplas I see. no problem, commit your suggestions and resolve errors (my bad).